api/user/user.model.js

```diff
@@ -1,8 +1,28 @@
 const mongoose = require('mongoose');
+const bcrypt = require('bcrypt-nodejs');
+const Promise = require('bluebird');
 
-const userSchema = new mongoose.Schema({
+
+const userSchema = mongoose.Schema({
```

Comment:
Does this pass the linter? I think we want to use keyword new to construct the new Schema.